stack-packs/packs/wordpress.js

```diff
@@ -17,28 +17,28 @@ module.exports = {
+      'There a number of WordPress plugins that can help you [inline critical assets](https://wordpress.org/plugins/search/critical+css/) or [defer less important resources](https://wordpress.org/plugins/search/defer+css+javascript/). Beware that optimizations provided by these plugins may break features of your theme or plugins, so you will likely need to make code changes.',
     'unminified-css':
-      'A number of WordPress plugins can speed up your site by concatenating, minifying, and compressing your styles. You may also want to use a build process to do this minification up-front if possible.',
+      'A number of [WordPress plugins](https://wordpress.org/plugins/search/minify+css/) can speed up your site by concatenating, minifying, and compressing your styles. You may also want to use a build process to do this minification up-front if possible.',
     'unminified-javascript':
-      'A number of WordPress plugins can speed up your site by concatenating, minifying, and compressing your scripts. You may also want to use a build process to do this minification up-front if possible',
+      'A number of [WordPress plugins](https://wordpress.org/plugins/search/minify+javascript/) can speed up your site by concatenating, minifying, and compressing your scripts. You may also want to use a build process to do this minification up-front if possible',
     'efficient-animated-content':
-      'Consider uploading your GIF to a service which will then make it available to embed an HTML5 video. Alternatively find a plugin that will automatically convert GIF images into videos upon upload.',
+      'Consider uploading your GIF to a service which will then make it available to embed an HTML5 video.',
     'unused-javascript':
       'Consider reducing, or switching, the number of [WordPress plugins](https://wordpress.org/plugins/) loading unused JavaScript in your page. To identify plugins that are adding extraneous JS, try running [code coverage](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) in Chrome DevTools. You can identify the theme/plugin responsible from the URL of the script. Look out for plugins that have many scripts in the list which have a lot of red in code coverage. A plugin should only enqueue a script if it is actually used on the page.',
     'uses-long-cache-ttl':
-      'Read more about [Browser Caching in WordPress](https://codex.wordpress.org/WordPress_Optimization#Browser_Caching).',
+      'Read about [Browser Caching in WordPress](https://codex.wordpress.org/WordPress_Optimization#Browser_Caching).',
     'uses-optimized-images':
-      'Consider using a WordPress plugin that optimizes your images, automatically serves the required sizes, and serves them from a CDN.',
+      'Consider using a [WordPress plugin](https://wordpress.org/plugins/search/optimize+images/) that optimizes your images, automatically serves the required sizes, and serves them from a CDN.',
```

Comment:
```suggestion
      'Consider using an [image optimization WordPress plugin](https://wordpress.org/plugins/search/optimize+images/) that compresses your images while retaining quality.',
```

just a fan of link text being descriptive ;)